projects/ngx-drag-to-select/src/lib/operators.ts

```diff
@@ -1,14 +1,16 @@
-import { Observable } from 'rxjs/Observable';
+import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { SelectBox, SelectBoxInput, SelectContainerHost } from './models';
+import { SelectBox, SelectBoxInput, SelectContainerHost, MousePosition } from './models';
 import { getRelativeMousePosition } from './utils';
 
 export const createSelectBox = (container: SelectContainerHost) => (
   source: Observable<SelectBoxInput>
 ): Observable<SelectBox<number>> =>
   source.pipe(
     map(([event, opacity, { x, y }]) => {
-      const mousePosition = getRelativeMousePosition(event, container);
+      // Explicit cast to `MousePosition` in order to "use" `MousePosition`
+      // we have to import `MousePosition` otherwise we'll get a build error
+      const mousePosition: MousePosition = getRelativeMousePosition(event, container);
```

Comment:
Do you got a better solution?
The build was throwing an error because it didn't know `MousePosition`.

Reply:
And this doesn't work if we add the return type to `getRelativeMousePosition`?

Reply:
I'll try it out later today.

Reply:
I did a quick check and it doesn't work.
I added a return type to `getMousePosition` because `getRelativeMousePosition` already has a return type?

Reply:
Yep, indeed. This makes sense. The compiler is trying to figure out the shape of `MouseEvent` but it can't cause it's defined in a different module and never imported to the `operators.ts` module. So we can just add the type annotation and remove the comment I guess. What do you think? Or should we keep the comment?

Reply:
I'm afraid future us will try to remove it 😅 and bumping into this problem again.

Reply:
Haha you're right, we can leave it in then. Maybe let's reword this to `Type annotation is required here, because it implicitly uses MousePosition can the TS compiler has no idea what about the shape of this type.`, or something along those lines 🙂